build-configs.yaml

```diff
@@ -471,19 +471,14 @@ build_configs:
     branch: 'android-4.19-stable'
     variants: *android_variants
 
-  android_4.19:
+  android11_5.4:
```

Comment:
The convention for all the other entries is `tree-name_branch-or-short-name`.  So `android_11-5.4` would be more consistent.  Not a big deal though, but I thought it was worth mentioning.

Reply:
Fixed in #434.